canonical-websites/cn.ubuntu.com
# Implement blog on cn.ubuntu.com

## Done

- Implements blog-module
- Updates the templates

Fixes https://github.com/ubuntudesign/base-squad/issues/427
Fixes https://github.com/ubuntudesign/base-squad/issues/426

## QA

- `./run`
- navigate to `http://localhost:8010`, make sure everything loads correctly
- navigate to `http://localhost:8010/blog`, all posts should appear. No filtering for chinese posts is active yet
